scripts/importer/data_importer/raw_data_importer.py

```diff
@@ -264,18 +264,19 @@ def _insert_variants(self):
                     except KeyError:
                         hom_counts = None # null is better than 0, as 0 has a meaning
                     except ValueError:
-                        data['hom_count'] = tuple(int(count) for count in info['AC_Hom'].split(',')) # parsing Swegen sometimes give e.g. 14,0
+                        hom_count = tuple(int(count) for count in info['AC_Hom'].split(','))  # parsing Swegen sometimes give e.g. 14,0
 
-                    data['orig_alt_alleles'] = [
-                        '{}-{}-{}-{}'.format(data['chrom'], *get_minimal_representation(base['pos'], base['ref'], x)) for x in alt_alleles
-                    ]
 
                     for i, alt in enumerate(alt_alleles):
                         if not self.settings.beacon_only:
                             vep_annotations = [ann for ann in annotations if int(ann['ALLELE_NUM']) == i + 1]
 
                         data = dict(base)
                         data['pos'], data['ref'], data['alt'] = get_minimal_representation(base['pos'], base['ref'], alt)
+                        data['orig_alt_alleles'] = [
+                            '{}-{}-{}-{}'.format(data['chrom'], *get_minimal_representation(base['pos'], base['ref'], x)) for x in alt_alleles
```

Comment:
An identical copy will be recalculated every time. Keep calculation outside for loop, assignment here?